pkg/hostagent/snatlocalinfo.go

```diff
@@ -32,6 +32,30 @@ type SnatLocalInfo struct {
 	snatpolicyName string
 }
 
+func (agent *HostAgent) populateSnatLocalInfos() error {
+	env := agent.env.(*K8sEnvironment)
+	snatLocalInfoClient := env.snatLocalInfoClient
+	snatLocalInfoCr, err := snatLocalInfoClient.AciV1().SnatLocalInfos(agent.config.AciSnatNamespace).Get(context.TODO(), agent.config.NodeName, metav1.GetOptions{})
+	if err != nil {
+		agent.log.Error("Failed to get snatlocalinfo ", err.Error())
+		return err
+	}
+	for _, localInfo := range snatLocalInfoCr.Spec.LocalInfos {
+		if localInfo.PodUid != "" {
+			agent.indexMutex.Lock()
+			_, ok := agent.opflexSnatLocalInfos[localInfo.PodUid]
```

Comment:
nit: I don't think step is necessary, but don't need to change if everything else is working

Reply:
addressed